### DISCUSSION TITLE
[Discord] World-alias placeholder support

### DISCUSSION THREAD

**Discussion by HcNguyen111** [EssentialsX/Essentials#4416] [Ideas and Feature Suggestions] [answered]
EssentialsX currently supports world-aliases, and world names and can be used in chat formatting via the `{WORLD}` and `{WORLDNAME}` placeholders.

EssentialsDiscord only has `{WORLD}` which only returns the world name. "World-aliases" set in EssentialsX config.yml can't be returned at all.

Suggestion: add the ability to use world-aliases set in EssentialsX in EssentialsDiscord - specifically in `mc-to-discord`, by maybe adding the same placeholders:
```
# {WORLD} - The world alias of the sender's current world. See the world-aliases section below for details.
# {WORLDNAME} - The full name of the sender's current world.
# {SHORTWORLDNAME} - The first character of the sender's current world.
```

**Comment by Chew** [comment]
EssentialsDiscord supports [PlaceholderAPI](https://www.spigotmc.org/resources/placeholderapi.6245), and with the [Multiverse-Core Expansion](https://github.com/PlaceholderAPI/PlaceholderAPI/wiki/Placeholders#multiverse-core), you can get the desired effect.

**Reply by HcNguyen111** [reply]
That is true, though would require Multivere-Core (which does much more than just world-aliases) to be "installed" - which is overkill if only world-alias support is the goal - as in my case.

[Opinion] This suggestion, if implemented, would tie EssentialsX with EssentialsDiscord a bit more completely as it already exists in EssentialsX.

**Comment by JRoy** [comment, marked as answer]
Will be added in https://github.com/EssentialsX/Essentials/pull/4419
